plugins/foreman_openscap/0.7/index.md

```diff
@@ -343,6 +343,78 @@ If you find a bug, please file it in
 See the [troubleshooting section](/manuals/latest/index.html#7.2GettingHelp)
 in the Foreman manual for more info.
 
+### 7.1.1 Debugging foreman_scap_client
+
+Everything seems to be set up correctly but the reports are not showing up in UI? These are a few debugging steps:
+
+
+* Running with Katello
+
+When running with [Katello](/plugins/katello/index.html), make sure your host is registered as [Content Host](plugins/katello/nightly/user_guide/content_hosts/index.html). Consumer certificates are used for client authentication instead of Puppet certs.
+
+
+* Scans based on cronlines
+
+Scanning and report generation are based on cron. You can inspect all the cron lines in ```/etc/cron.d/foreman_scap_client_cron```. This file is managed by Puppet and any changes will be overwritten on next Puppet run.
+
+
+* Config for client
+
+Config file for foreman_scap_client is located at ```/etc/foreman_scap_client/config.yaml```. It should look something like this:
+
+
+    # DO NOT EDIT THIS FILE MANUALLY
+    # IT IS MANAGED BY PUPPET
+
+    # Foreman proxy to which reports should be uploaded
+    :server: 'somewhere.example.com'
+    :port: 8443
+
+    ## SSL specific options ##
+    # Client CA file.
+    # It could be Puppet CA certificate (e.g., '/var/lib/puppet/ssl/certs/ca.pem')
+    # Or (recommended for client reporting to Katello) subscription manager CA file, (e.g., '/etc/rhsm/ca/katello-server-ca.pem')
+    :ca_file: '/etc/puppetlabs/puppet/ssl/certs/ca.pem'
+    # Client host certificate.
+    # It could be Puppet agent host certificate (e.g., '/var/lib/puppet/ssl/certs/myhost.example.com.pem')
+    # Or (recommended for client reporting to Katello) consumer certificate (e.g., '/etc/pki/consumer/cert.pem')
+    :host_certificate: '/etc/puppetlabs/puppet/ssl/certs/ada-bivens.example.com.pem'
+    # Client private key
+    # It could be Puppet agent private key (e.g., '/var/lib/puppet/ssl/private_keys/myhost.example.com.pem')
+    # Or (recommended for client reporting to Katello) consumer private key (e.g., '/etc/pki/consumer/key.pem')
+    :host_private_key: '/etc/puppetlabs/puppet/ssl/private_keys/ada-bivens.example.com.pem'
+
+    # policy (key is id as in Foreman)
+
+    1:
+      :profile: 'xccdf_org.ssgproject.content_profile_stig-java-upstream'
+      :content_path: '/var/lib/openscap/content/fe93f99c14251cc76e92b9da71c351c8ba45fbd3639a2cd55911ef6f7db1b650.xml'
+      # Download path
+      # A path to download SCAP content from proxy
+      :download_path: '/compliance/policies/1/content/fe93f99c14251cc76e92b9da71c351c8ba45fbd3639a2cd55911ef6f7db1b650'
+      :tailoring_path: ''
+      :tailoring_download_path: ''
+
+There will be an entry for each policy that is assigned to your host.
```

Comment:
The policy entry starts by number followed by colon. Policy attributes are indented by 2 spaces.